recipes/xrandr.lua

```diff
@@ -0,0 +1,146 @@
+--- Separating Multiple Monitor functions as a separeted module (taken from awesome wiki)
+
+local awful     = require("awful")
+local naughty   = require("naughty")
+local beautiful = require("beautiful");
+
+-- A path to a fancy icon
+local icon_path = "";
+
+-- Get active outputs
+local function outputs()
+   local outputs = {}
+   local xrandr = io.popen("xrandr -q")
+
+   if xrandr then
+      for line in xrandr:lines() do
+         local output = line:match("^([%w-]+) connected ")
+         if output then
+            outputs[#outputs + 1] = output
+                                   end
+      end
+      xrandr:close()
```

Comment:
I guess so (even though I really, really hate parsing xrandr output. For example, there is a Java library that does so internally and it broke on wayland for $WEIRD_REASONS.)